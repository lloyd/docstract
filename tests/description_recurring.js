/**
 * Description may recur.  lots of times.
 *
 * @desc Here's a little bit.
 * @description
 * Here's a little bit more.
 *
 * @desc and how bout a final bit of module desc?
 */

/**
 * @function foo
 * The function that's documented all over because in functions...
 * @param {string} bar An AWESOME param. 
 * @description
 * ..description can be interspersed...
 * @param {string} baz Another AWESOME param.
 * @desc between other tags!
 */
