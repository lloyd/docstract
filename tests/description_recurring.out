{
  "desc": "Description may recur.  lots of times.\n\nHere's a little bit.\n\nHere's a little bit more.\n\nand how bout a final bit of module desc?", 
  "filename": "tests/description_recurring.js", 
  "functions": [
    {
      "desc": "The function that's documented all over because in functions...\n\n..description can be interspersed...\n\nbetween other tags!", 
      "name": "foo", 
      "params": [
        {
          "desc": "An AWESOME param.", 
          "name": "bar", 
          "type": "string"
        }, 
        {
          "desc": "Another AWESOME param.", 
          "name": "baz", 
          "type": "string"
        }
      ], 
      "source_lines": [
        11, 
        19
      ]
    }
  ], 
  "module": "description_recurring", 
  "source_lines": [
    1, 
    10
  ]
}
